### DISCUSSION TITLE
Feature Request - Dynamic tag settings for Lightbox links to pull from custom fields

### DISCUSSION THREAD

**Discussion by portrayaloflife** [elementor/elementor#21455] [Feature Request]
### Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

The ability to pull from a custom field for the Lightbox dynamic tag. We've had several projects where in a custom loop we need a video component and having the ability for a custom field to populate the video URL so they can open in lightbox would be amazing!

### Describe the solution you'd like

Just like the "Post Custom Field" dynamic tag, adding the ability for lightbox to have settings where the Video URL could be pulled from a custom field! 

### Describe alternatives you've considered

hardcoding it

**Comment by portrayaloflife** [comment]
This would allow folks to build custom loops with video components. Could even do video galleries now, where a custom loop could be a client setting the feature image, with a play icon over the image that pulls a youtube URL from a custom field that populates the lightbox link!

**Comment by jmcintyresage** [comment]
I need this feature too -  in my case, to wire up a "Watch" button for videos whose URLs are saved in an ACF custom field. I considered hardcoding briefly, but that seemed difficult and potentially fragile.

**Comment by DevAndLetDie** [comment]
@nicholaszein Is this possible? Could we pay to get this implemented?
